matrix_client/client.py

```diff
@@ -137,6 +142,7 @@ def __init__(self, base_url, token=None, user_id=None,
         self.ephemeral_listeners = []
         self.device_id = None
         self._encryption = encryption
+        self.encryption_conf = encryption_conf or {}
```

Comment:
tricksy one-liner there. Had to double-check in REPL to make sure it did what I expected.